clustered/common/src/main/java/org/ehcache/clustered/common/messages/ResponseCodec.java

```diff
@@ -94,6 +96,15 @@
         buffer.putInt(((ClientInvalidateAll) response).getInvalidationId());
         return buffer.array();
       }
+      case SERVER_INVALIDATE_HASH: {
+        ServerInvalidateHash serverInvalidateHash = (ServerInvalidateHash) response;
+        byte[] cacheIdBytes = serverInvalidateHash.getCacheId().getBytes(Charset.forName("UTF-8"));
```

Comment:
We have inconsistent cache id encoding between this class and `ServerStoreOpCodec`. Would be good to pick one and only one.